### DISCUSSION TITLE
BTT TFT documentation

### DISCUSSION THREAD

**Discussion by bttguy** [bigtreetech/BIGTREETECH-TouchScreenFirmware#1711] [Ideas]
I would like to ask what do you think about the idea to use the Wiki tab, to organise the growing documentation of this evolving project. I am not sure if it is needed to maintained by bigtreetech, or the community could also help, but a nicely organised Wiki could be more helpful than the one page with every possible information.
If it would be insustainible, its just a quick idea, saw some other repos with a nice Wiki page :)

**Comment by oldman4U** [comment]
This would be great. Could you please think about a possible structure like a menu table and present it here.

Thank you

**Comment by bttguy** [comment]
@bigtreetech What do you think? How can a wiki page be managed? If we can sort this out, I can start thinking about its structure!

**Reply by bigtreetech** [reply]
I don't know much about wikis. If you are willing to help me understand how to use good wiki, I would be happy and thank you very much!

**Reply by bttguy** [reply]
Try to enable wikis, than go to settings and under feature settings untick restrict editing. Than I will see if I can edit the wiki, and what we can do further. I am also new to this topic but this is where to start I think.

Step by step:
![wiki](https://user-images.githubusercontent.com/76131447/111923988-647a3e00-8aa2-11eb-8646-2043d6c442e6.png)
![wiki2](https://user-images.githubusercontent.com/76131447/111923989-6512d480-8aa2-11eb-866e-68836b2449b2.png)
![wiki3](https://user-images.githubusercontent.com/76131447/111923990-6512d480-8aa2-11eb-9ca4-8ac00aeea4a8.png)
![wiki4](https://user-images.githubusercontent.com/76131447/111923991-6512d480-8aa2-11eb-9166-c573dd13700c.png)
![ki5](https://user-images.githubusercontent.com/76131447/111923992-65ab6b00-8aa2-11eb-954d-b0d126c42a2a.png)

**Reply by bttguy** [reply]
After you opened it and let the users edit/contribute, I can see how I can edit irl and can start to migrate the current content of the readme, and you can see if this is any good. If you/the community dont like it, you can just delete the wiki :) I also dont want to spend a lot of time researching how is it working exactly when it is just an idea, but I can/willing to learn it by/as doing it if this is good for you :) And I am also sure if it is any good others also could help who already contributed a lot into the readme documentation.

For reference, this is a nice looking Wiki: https://github.com/Duet3D/RepRapFirmware/wiki

**Comment by bttguy** [comment]
@bigtreetech Hi! I saw you enabled the wiki page, but it's not editable yet. In the settings you should untick restrict editing to collaborators only.

**Comment by 9zdot** [comment]
Very good initiative. 
Lack of documentation really slow down on what community can contribute once each one needs to do a reverse engineering before produce something.
